app/client/components/user/EditUserForm.tsx

```diff
@@ -33,6 +33,8 @@ export interface IEditUser {
   bio?: string;
   referredBy?: string;
   location?: string;
+  latitude?: number;
```

Comment:
forgive my ignorance; does `number` cover `float` too?

Reply:
Yes it does.
[Number in typescript](https://www.typescriptlang.org/docs/handbook/basic-types.html#number)